lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -151,6 +151,12 @@ def enqueue_delayed(klass, *args)
         end
       end
 
+      # Given an encoded item, remove it from the queue
```

Comment:
Could you improve this comment to clearly indicate that the magic here is when the default queue doesn't match the queue that the job is in?